src/System.Globalization/tests/Performance/Perf.StringCultureInfo.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
They don't exist in the .NET Core surface area:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L1588-L1589